lib/tasks/create-development-user.rake

```diff
@@ -0,0 +1,7 @@
+namespace :dev do
+  desc 'Create a user to avoid GDS SSO in development - not for production'
+  task :create_user => :environment do
+    puts "Creating a single user if it doesn't already exist"
```

Comment:
Maybe should put a test to check the RAILS_ENV is not production here, just in case?